ui/packages/codice-icons/codice.font.css

```diff
@@ -0,0 +1,3 @@
+.cf:before {
+  vertical-align: middle !important;
```

Comment:
Consider leaving off the `!important`, not sure why but maybe someone will want to override this.

Reply:
changed 👍